resources/assets/lib/chat.ts

```diff
@@ -35,28 +34,26 @@ reactTurbolinks.register('chat', MainView, () => {
   const sendTo: SendToJSON = osu.parseJson('json-sendto');
 
   if (!_.isEmpty(sendTo)) {
-    const target: User = dataStore.userStore.getOrCreate(sendTo.target.id, sendTo.target); // pre-populate userStore with target
-    let channel: Channel | null = dataStore.channelStore.findPM(target.id);
+    const target = dataStore.userStore.getOrCreate(sendTo.target.id, sendTo.target); // pre-populate userStore with target
+    let channel = dataStore.channelStore.findPM(target.id);
 
     if (channel) {
       initialChannel = channel.channelId;
-    } else {
+    } else if (!target.is(sendTo.target)) {
```

Comment:
Not sure in what case `target` - which is from `getOrCreate(...id, sendTo.target)` - is not `sendTo.target` ?_?